cluster/images/etcd-version-monitor/etcd-version-monitor.go

```diff
@@ -1,5 +1,6 @@
 /*
 Copyright 2017 The Kubernetes Authors.
+Copyright 2020 Authors of Arktos - file modified.
```

Comment:
Is this file only has copyright change? Does not look right

Reply:
I ran make update and got it.